tests/team.test.js

```diff
@@ -354,6 +354,102 @@ describe("PATCH change team", function () {
                     res.body.message.should.equal(Constants.Success.TEAM_JOIN);
                     res.body.should.have.property("data");
 
+                    done();
+                });
+        });
+    });
+});
+
+describe("PATCH change team info", function () {
+    it("should FAIL to change a hacker's team information", function (done) {
```

Comment:
Describe failure case: "should FAIL to change a hacker's team information due to invalid authentication"